apps/moderation/src/components/content-list.tsx

```diff
@@ -250,28 +215,27 @@ const ContentList: React.FC<IContentListProps & RootComponentProps> = props => {
                         : ''
                     }
                     reportedOnLabel={t('On')}
-                    reportedDateTime={pendingItem.entryDate}
+                    reportedDateTime={pendingItem.reportedDate}
                     makeADecisionLabel={t('Make a Decision')}
                     handleButtonClick={handleButtonClick}
                   />
                 ))}
               </Box>
             ))}
-            {/* fetch indicator for load more on scroll */}
-            {pendingItemsQuery.isLoading && pendingItemPages.length < 1 && (
-              <Box pad="large">
-                <Spinner />
-              </Box>
-            )}
             {/* triggers intersection observer */}
             <Box pad="xxsmall" ref={loadmorePendingRef} />
           </>
         ) : (
           <NoItemsFound activeTab={'pending'} />
         ))}
-      {!delistedItemsQuery.isLoading &&
-        !isPending &&
-        (isDelisted && delistedItemPages.length ? (
+      {/* fetch indicator for load more on scroll */}
+      {pendingItemsQuery.isLoading && isPending && (
+        <Box pad="large">
+          <Spinner />
+        </Box>
+      )}
+      {!isPending &&
+        (!delistedItemsQuery.isLoading && isDelisted && delistedItemPages.length ? (
```

Comment:
it is better to reassign these expressions to a more verbose var(for readability) and could also be wrapped with `useMemo`